eventmesh-storage-plugin/eventmesh-storage-rocketmq/src/main/java/org/apache/eventmesh/storage/rocketmq/producer/ProducerImpl.java

```diff
@@ -68,7 +68,7 @@ public void setExtFields() {
         super.getRocketmqProducer().setCompressMsgBodyOverHowmuch(10);
     }
 
-    public SendResult send(CloudEvent cloudEvent) {
+    public SendResult send(CloudEvent cloudEvent) throws InterruptedException {
```

Comment:
Your modification is not much different from the original code, because the `catch` here does not throw `InterruptedException`, so it still swallows `InterruptedException`. What I mean is to throw `InterruptedException` directly in `catch(InterruptedException)` or return empty `SendResult`  in `catch(InterruptedException)`.

您这样修改和原代码区别不大，因为这里的`catch`不会抛出`InterruptedException`，所以还是将`InterruptedException`吞掉了。我的意思是在`catch(InterruptedException)`中直接抛出InterruptedException或者返回空的`SendResult`。

Reply:
I'm very sorry again. QAQ